packages/@aws-cdk/lambda-layer-kubectl/lib/kubectl-layer.ts

```diff
@@ -1,27 +1,32 @@
-import * as crypto from 'crypto';
-import * as fs from 'fs';
+// import * as crypto from 'crypto';
+// import * as fs from 'fs';
 import * as path from 'path';
 import * as lambda from '@aws-cdk/aws-lambda';
 import { Construct } from 'constructs';
 
+/**
+ * KubectlLayerProps 
+ */
+export interface KubectlLayerProps {
+  /**
+   * Kubectl Version which is actually a version of EKS cluster.
+   */
+  readonly kubectlVersion: string;
+}
+
 /**
  * An AWS Lambda layer that includes `kubectl` and `helm`.
  */
 export class KubectlLayer extends lambda.LayerVersion {
-  constructor(scope: Construct, id: string) {
+  constructor(scope: Construct, id: string, props: KubectlLayerProps) {
```

Comment:
```suggestion
  constructor(scope: Construct, id: string, props?: KubectlLayerProps) {
```
We probably should make it optional for backward compatibility?

Reply:
Agreed.